tests/conftest.py

```diff
@@ -185,3 +185,47 @@ def build(self, **kwargs):
             return membership
 
     return MembershipFactory()
+
+@pytest.fixture
+@pytest.mark.django_db
+def subject(db, bill):
+    class SubjectFactory():
+        def build(self, **kwargs):
+            related_bill = bill.build()
+            subject_name = 'Metro Gold Line'
+
+            subject_info = {
+                'id': randrange(10000),
+                'bill': related_bill,
+                'subject': subject_name
+            }
+
+            subject_info.update(kwargs)
+
+            subject = Subject.objects.create(**subject_info)
+
+            return subject
+
+    return SubjectFactory()
+
+@pytest.fixture
+@pytest.mark.django_db
+def subject_guid(db, subject):
+    class SubjectGuidFactory():
+        def build(self, **kwargs):
+            related_subject = subject.build()
+            guid = '0000-0-0000'
+
+            subject_guid_info = {
+                'id': randrange(10000),
```

Comment:
just popping in: django will set the id for you, so you don't need to set one here. 👌